src/Docfx.App/RunServe.cs

```diff
@@ -44,6 +48,7 @@ public static void Exec(string folder, string host, int? port, bool openBrowser,
             Console.WriteLine($"Serving \"{folder}\" on {url}");
             Console.WriteLine("Press Ctrl+C to shut down");
             using var app = builder.Build();
+            app.UseExtensionlessHtmlUrl();
```

Comment:
Should this be opt-in? I think the default behavior should remain a standard static file server so it can be hosted by a variety of web servers.

Reply:
This settings is works on `docfx serve` command only.
And not affect generated HTML files.

So I though it's not a problems by enabled default. (And not support config to disable)
When supporting feature that rewrite URL to `extensionless URL` automatically.
It should be supported as user's opt-in features.